src/wallet-user/ContextualizedWallet.tsx

```diff
@@ -31,89 +33,107 @@ export default function ContextualizedWallet() {
-                    },
-                    onClick: refreshAll,
-                    disabled: !vaultState,
+                background: colorTheme.topMenuItems.iconGradient,
+            },
+            onClick: refreshAll,
+        },
+        {
+            id: "vault",
+            text: "Vault",
+            to: VAULT_PATH,
+            exact: false,
+            icon: {
+                icon: {
+                    id: 'vault'
                 },
-                {
-                    id: "loc-collection",
-                    text: "Collections",
-                    to: locRequestsPath('Collection'),
-                    exact: false,
-                    icon: {
-                        icon: {
-                            id: 'collection'
-                        },
-                        background: colorTheme.topMenuItems.iconGradient,
-                    },
-                    onClick: refreshAll,
+                background: colorTheme.topMenuItems.iconGradient,
+            },
+            onClick: refreshAll,
+            disabled: !vaultState,
+        },
+        {
+            id: "loc-collection",
+            text: "Collections",
+            to: locRequestsPath('Collection'),
+            exact: false,
+            icon: {
+                icon: {
+                    id: 'collection'
                 },
-                {
-                    id: "loc-transaction",
-                    text: "Transactions",
-                    to: locRequestsPath('Transaction'),
-                    exact: false,
-                    icon: {
-                        icon: {
-                            id: 'loc'
-                        },
-                        background: colorTheme.topMenuItems.iconGradient,
-                    },
-                    onClick: refreshAll,
+                background: colorTheme.topMenuItems.iconGradient,
+            },
+            onClick: refreshAll,
+        },
+        {
+            id: "loc-transaction",
+            text: "Transactions",
+            to: locRequestsPath('Transaction'),
+            exact: false,
+            icon: {
+                icon: {
+                    id: 'loc'
                 },
-                {
-                    id: "loc-identity",
-                    text: "Identity",
-                    to: locRequestsPath('Identity'),
-                    exact: false,
+                background: colorTheme.topMenuItems.iconGradient,
+            },
+            onClick: refreshAll,
+        },
+        {
+            id: "loc-identity",
+            text: "Identity",
+            to: locRequestsPath('Identity'),
+            exact: false,
+            icon: {
+                icon: {
+                    id: 'identity'
+                },
+                background: colorTheme.topMenuItems.iconGradient,
+            },
+            onClick: refreshAll,
+        },
+    ];
+
+    if (locsState?.isVerifiedThirdParty) {
```

Comment:
`locsState` may be discarded.

Reply:
Fixed in https://github.com/logion-network/logion-wallet/pull/392/commits/ed54b9680fbb6815fc5e91cbf532e3c3a67ffd14